toolchain/gcc/PKGBUILD

```diff
@@ -121,11 +121,17 @@ build() {
   export CXXLAGS+=" -fno-semantic-interposition"
 
   if [ -n "$_optimize_v3" ]; then
-    export ASFLAGS="-D__AVX__=1 -D__AVX2__=1 -msse2avx -D__FMA__=1 "
-  elif [-n "$_optimize_v4" ]; then
-    export ASFLAGS="-D__AVX__=1 -D__AVX2__=1 -D__AVX512__=1 -msse2avx -D__FMA__=1 "
+    export ASFLAGS+="-D__AVX__=1 -D__AVX2__=1 -msse2avx -D__FMA__=1 "
+    export LDFLAGS+=" -Wl,-z,max-page-size=0x1000"
```

Comment:
What is the point of explicitly setting max-page-size? 0x1000 (4kb page) is default value for x86. This would make sense to change if our package had a build condition under ARM arch, where the page size can be changed.

Reply:
Well, this was cherry picked too from clearlinux. I know, that the page size is default to 4kb.

Maybe glibc/gcc doing some internal things, and make it bigger/smaller here and there? Questionable.
I think having it in wont hurt at all

Reply:
Yes, I thought about that too - but did not do any tests yet. Im afraid, that this increases generally the memory usage massively.